.github/phpstan.neon

```
@@ -0,0 +1,32 @@
+            - '#Call to an undefined method Kitodo\\Dlf\\Domain\\Repository\\[a-zA-Z]+Repository::findOneByRoot\(\)\.#'
+            - '#Call to an undefined method Kitodo\\Dlf\\Domain\\Repository\\[a-zA-Z]+Repository::findOneBySessionId\(\)\.#'
+            - '#Call to an undefined method Kitodo\\Dlf\\Domain\\Repository\\[a-zA-Z]+Repository::findOneByType\(\)\.#'
+            - '#Call to an undefined method Kitodo\\Dlf\\Domain\\Repository\\[a-zA-Z]+Repository::findOneByUid\(\)\.#'
+            - '#Call to an undefined method Psr\\Http\\Message\\RequestFactoryInterface::request\(\)\.#'
+            - '#Call to an undefined method Solarium\\Core\\Query\\DocumentInterface::setField\(\)\.#'
+            - '#Call to an undefined method Ubl\\Iiif\\Presentation\\Common\\Model\\Resources\\IiifResourceInterface::getHeight\(\)\.#'
+            - '#Call to an undefined method Ubl\\Iiif\\Presentation\\Common\\Model\\Resources\\IiifResourceInterface::getWidth\(\)\.#'
+            - '#Call to an undefined method Ubl\\Iiif\\Presentation\\Common\\Model\\Resources\\IiifResourceInterface::getPossibleTextAnnotationContainers\(\)\.#'
+            - '#Call to an undefined method Ubl\\Iiif\\Presentation\\Common\\Model\\Resources\\IiifResourceInterface::getTextAnnotations\(\)\.#'
+            - '#Call to an undefined method Ubl\\Iiif\\Presentation\\Common\\Model\\Resources\\ManifestInterface::getOriginalJsonArray\(\)\.#'
+            - '#Call to an undefined method Ubl\\Iiif\\Presentation\\Common\\Model\\Resources\\RangeInterface::getMemberRangesAndRanges\(\)\.#'
+            - '#Constant LOG_SEVERITY_ERROR not found\.#'
+            - '#Constant LOG_SEVERITY_NOTICE not found\.#'
+            - '#Constant LOG_SEVERITY_WARNING not found\.#'
+            - '#Constant TYPO3_MODE not found\.#'
+            - '#Method Kitodo\\Dlf\\Controller\\Backend\\NewTenantController::errorAction\(\) should return Psr\\Http\\Message\\ResponseInterface but return statement is missing\.#'
```

This should be fixed in the code instead of ignoring the error message. But maybe this is just a side-effect of the recursion from #1045 (see below).
```suggestion
```

---

Yeah, I was changing there to ResponseInterface, but as you good pointed out currently we are not returning there anything so we should ignore this error.

---

https://github.com/kitodo/kitodo-presentation/pull/1052#discussion_r1368926320